src/ansys_sphinx_theme/__init__.py

```diff
@@ -24,10 +24,11 @@
 import logging
 import os
 import pathlib
+import subprocess
 from typing import Any, Dict
 
 from docutils.nodes import document
-import requests
+import pymupdf
```

Comment:
Have we verified that we can use this pacakge? It's license is GNU AFFERO GPL 3.0. I'm guessing there is no problem but I am also interested in its approval for other libraries.

Reply:
https://www.gnu.org/licenses/agpl-3.0.html

Reply:
I do not think this is an issue for other libraries since they use the theme for development. However, it is an issue within this project for building the wheelhouse.

Reply:
Yep - that was my concern. Redistributing is not well seen with AGPL 3.0 from waht I understood, and the wheelhouse is a clear redistribution of software...

Reply:
Thank you for raising the issue, I initially used this approach to avoid adding extra dependencies on ImageMagick or Poppler. However, given the issues we encountered, I reverted to using pdf2image and added a note explaining the change. Please review